src/webserver/Listener.h

```diff
@@ -51,7 +51,7 @@ class Detector
     std::shared_ptr<ETLLoadBalancer> balancer_;
     std::shared_ptr<ReportingETL const> etl_;
     util::TagDecoratorFactory const& tagFactory_;
-    DOSGuard& dosGuard_;
+    clio::DOSGuard& dosGuard_;
```

Comment:
it is kinda weird , we need to use clio:: to use something in the same folder. If I understand correctly, you also plan to put all these class under clio:: , right?

Reply:
We can reconsider this.. i don't mind having to type `clio::` in headers tho. and for implementation files you can always do `using namespace clio;`.
Ideally it would eventually all be in one namespace `clio`. When that's done you will not need to type `clio::` anymore as we will be in the same namespace already.